### ISSUE TITLE
Als medewerker belastingzaken en verantwoordelijk bronhouder wil ik de WOZ waarden en alle beschikbare WOZ detailgegevens van objecten in mijn gemeente kunnen opzoeken en inzien

### ISSUE THREAD

**Issue by CathyDingemanse** [VNG-Realisatie/Haal-Centraal-WOZ-bevragen#7]
zodat ik bij outsourcing van deze taak aan een samenwerkingsverband te aller tijde over de gegevens kan beschikken waarvoor ik eindverantwoordelijk ben.

**Comment by fsamwel** [comment]
@CathyDingemanse bedoel je met "alle beschikbare WOZ detailgegevens" echt letterlijk alle gegevens uit de WOZ? Bijvoorbeeld ook alle registratiedatums, enz.

**Comment by CathyDingemanse** [comment]
Nee, alleen de gegevens die in de GOALS Canvas zijn opgenomen. Sommige daarvan zitten niet in de LV maar in de Dataland set.
